File: IOSwiftUI/Common/Messages/OneTimeCodes/BOOneTimeCodeListRequestModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using IOSwiftUI.Common.Messages.Base;

namespace IOSwiftUI.Common;

public class BOOneTimeCodeListRequestModel : RequestModel
{
    [Required]
    public int Count { get; set; }

    [Required]
    public int Start { get; set; }
}

File: IOSwiftUI/Backoffice.UI/src/screens/onetimecodes/models/BOOneTimeCodeListRequestModel.ts
import BaseRequestModel from "../../../common/models/BaseRequestModel";

class BOOneTimeCodeListRequestModel extends BaseRequestModel {

    count: number;
    start: number;

    constructor() {
        super();

        this.count = 0;
        this.start = 0;
    }
}

export default BOOneTimeCodeListRequestModel;
